pkg/api/metadata.go

```diff
@@ -69,14 +88,22 @@ func (m *Metadata) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 
 // ServeHTTP sends API version v1 documentation response.
 func (v *VersionMetadata) ServeHTTP(w http.ResponseWriter, r *http.Request) {
+	// Get the version:
+	version, err := getCommitSha()
+	if err != nil {
+		shared.HandleError(r, w, err.(*errors.ServiceError))
```

Comment:
the `err.(*errors.ServiceError)` cast assumes all errors from `getCommitSha` will be service error. It is fine for now since only the GeneralError is returned. To be on the safe side I'd have used `errors.ToServiceError(err)` here.